sirepo/sim_data/__init__.py

```diff
@@ -655,8 +655,12 @@ def _organize_example(cls, data):
             dm.simulation.folder = '/Examples'
 
 
-def uid_from_jid(jid):
-    return jid.split(_JOB_ID_SEP)[0]
+def split_jid(jid):
+    """Reverse of parse_jid. Take a jid and split it into its parts"""
```

Comment:
I struggle with comments for things like this, because it's obvious. It doesn't actually tell me what it returns. "parts" is too generic (tuple, list, dict?).  Also I think "simulation_id" is too long, and we used "sid" elsewhere.

```py
"""Split jid into named parts

Args:
    jid (str): properly formed job identifier
Returns:
    PKDict: parts named uid, sid, compute_model.
"""
```

Reply:
Makes sense. Thanks.

While we're on this road, why "properly formed"? Isn't that by definition? Is it to imply we don't do any checking and if you put malformed in you will get malformed out?

Reply:
https://github.com/radiasoft/sirepo/pull/2534/commits/9ce89be2fa5fb91607e6a20d55f8bf0cb9a69f45